src/main/java/apoc/load/Jdbc.java

```diff
@@ -43,13 +44,18 @@
     public Log log;
 
     private static Connection getConnection(String jdbcUrl) throws Exception {
-        URI uri = new URI(jdbcUrl.substring("jdbc:".length()));
-        String userInfo = uri.getUserInfo();
-        if (userInfo != null) {
-            String[] user = userInfo.split(":");
-            String cleanUrl = jdbcUrl.substring(0,jdbcUrl.indexOf("://")+3)+jdbcUrl.substring(jdbcUrl.indexOf("@")+1);
-            return DriverManager.getConnection(cleanUrl, user[0], user[1]);
+        if (!jdbcUrl.contains("#")) {
```

Comment:
I don't understand this check.

Reply:
User Info can also be in `scheme://user:password@host`

Reply:
Firstly, I'm sorry not to give you an answer sooner.

You are right, that check is not so understandable and it doesn't make a lot of sense, I pushed an old version of the correction. The issue happen when the jdbc connection string contains some special character as tested in the unit test Jdbc.testLoadJdbcWithSpecialChar .

In the last commit I just catch the URISyntaxException and in that case I skip the code that extract the user info from the URL, but I pass it directly to the driver. I know that neither this is a good solution because I'm thinking only now that the url can contain several kind of mistakes which are not related to this problem.  A more clean solution is needed I suppose.

Reply:
I think that the method getConnection shouldn't use the URI class to parser the jdbc url, because the jdbc driver accept characters that are not allowed in a normal URI. The Jdbc driver itself is parsing the parameter passed in order to validate it depending on the database used.

Reply:
Hi again, 
my code is making failing the CI but I didn't change anything else that could have broken 34 unit tests. Could you help me? The point is that we need to solve this bug in order to go on with my Company project. For now we're using our compiled jar, but in order to go to production, we need an official release.

Thank you in advance.
Stefano